### PR TITLE
{tools}[dummy/dummy] IntelClusterChecker v2017.1.016

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IntelClusterChecker/IntelClusterChecker-2017.1.016.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+description = """Verifies cluster components work together ― for 
+ - better uptime and productivity and lower total
+ - cost of ownership (TCO)
+ """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_clck_p_%(version)s.tgz']
+
+checksums = ['84e7c00bf1f9b87e248dd62e4df7e3b7']
```
Comment (boegel) [inline easybuild/easyconfigs/i/IntelClusterChecker/IntelClusterChecker-2017.1.016.eb]: @shahzebsiddiqui Do you mind making this a `SHA256` checksum? Also, no need for an empty line below `sources = ...`, makes sense to group these together.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2078.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8e5ef99892a8a6260707286561e7dd59 for a full test report.

File: easybuild/easyconfigs/i/IntelClusterChecker/IntelClusterChecker-2017.1.016.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+homepage = 'https://software.intel.com/en-us/intel-cluster-checker'
+description = """Verifies cluster components work together ― for 
+ - better uptime and productivity and lower total
+ - cost of ownership (TCO)
+ """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_clck_p_%(version)s.tgz']
+checksums = ['7a89553dd0bce18d6c17b14f94d669ffeef8621e01b0a135292fb7ce71fc6e25']
```
Comment (boegel) [inline easybuild/easyconfigs/i/IntelClusterChecker/IntelClusterChecker-2017.1.016.eb]: @shahzebsiddiqui Hmm, I'm getting a different SHA256 checksum: `72fcf16e220dfc1ee1e16a633381f7957e15b0b6f322f9954d1de2a1ec3851f1`.

The MD5sum is the same you had before though (`84e7c00bf1f9b87e248dd62e4df7e3b7 `)...

Comment (shahzebsiddiqui) [reply easybuild/easyconfigs/i/IntelClusterChecker/IntelClusterChecker-2017.1.016.eb]: okay let me fix not sure why there is a discrepancy in this. I downloaded this a long time ago

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2518.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3f9986c66011f2f5d2108add8fb3b948 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @shahzebsiddiqui!

**Comment by boegel** [comment]
Going in, thanks @shahzebsiddiqui!

**Comment by boegel** [comment]
Going in, thanks @shahzebsiddiqui!

**Pull request merged**